components/function-controller/internal/controllers/serverless/runtime/runtime.go

```diff
@@ -0,0 +1,73 @@
+package runtime
+
+import (
+	serverlessv1alpha1 "github.com/kyma-project/kyma/components/function-controller/pkg/apis/serverless/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+type Runtime interface {
+	SanitizeDependencies(dependencies string) string
+}
+
+type Config struct {
+	Runtime                 serverlessv1alpha1.Runtime
+	DependencyFile          string
+	FunctionFile            string
+	DockerfileConfigMapName string
+	RuntimeEnvs             []corev1.EnvVar
+}
+
+func GetRuntimeConfig(r serverlessv1alpha1.Runtime) Config {
+	switch r {
+	case serverlessv1alpha1.Nodejs12:
+		return Config{
+			Runtime:                 serverlessv1alpha1.Nodejs12,
+			DependencyFile:          "package.json",
+			FunctionFile:            "handler.js",
+			DockerfileConfigMapName: "dockerfile-nodejs-12",
+			RuntimeEnvs: []corev1.EnvVar{{Name: "NODE_PATH", Value: "$(KUBELESS_INSTALL_VOLUME)/node_modules"},
```

Comment:
Maybe it would be a good idea to set these values as a consts? (Speaking about `NODE_PATH`, `FUNC_RUNTIME` etc.)